files/ko/web/javascript/guide/expressions_and_operators/index.html

```diff
@@ -11,454 +11,721 @@
+      </td>
+      <td><code>3 === var1</code></td>
+    </tr>
+    <tr>
+      <td>
+        <a href="/ko/docs/Web/JavaScript/Reference/Operators/Strict_inequality">불일치</a>
+        (<code>!==</code>)
+      </td>
+      <td>피연산자의 값 또는 타입이 서로 다를 경우 <code>true</code>를 반환합니다.</td>
+      <td>
+        <code>var1 !== "3"<br />3 !== '3'</code>
+      </td>
+    </tr>
+    <tr>
+      <td>
+        <a href="/ko/docs/Web/JavaScript/Reference/Operators/Greater_than">큼</a>
+        (<code>&gt;</code>)
+      </td>
+      <td>왼쪽 피연산자가 오른쪽 피연산자보다 크면 <code>true</code>를 반환합니다.</td>
+      <td>
+        <code>var2 &gt; var1<br />"12" &gt; 2</code>
+      </td>
+    </tr>
+    <tr>
+      <td>
+        <a href="/ko/docs/Web/JavaScript/Reference/Operators/Greater_than_or_equal">크거나 같음</a>
+        (<code>&gt;=</code>)
+      </td>
+      <td>왼쪽 피연산자가 오른쪽 피연산자와 같거나 크면 <code>true</code>를 반환합니다.</td>
+      <td>
+        <code>var2 &gt;= var1<br />var1 &gt;= 3</code>
+      </td>
+    </tr>
+    <tr>
+      <td>
+        <a href="/ko/docs/Web/JavaScript/Reference/Operators/Less_than">작음</a>
+        (<code>&lt;</code>)
+      </td>
+      <td>왼쪽 피연산자가 오른쪽 피연산자보다 작으면 <code>true</code>를 반환합니다.</td>
+      <td>
+        <code>var1 &lt; var2<br />"2" &lt; 12</code>
+      </td>
+    </tr>
+    <tr>
+      <td>
+        <a href="/ko/docs/Web/JavaScript/Reference/Operators/Less_than_or_equal">작거나 같음</a>
+        (<code>&lt;=</code>)
+      </td>
+      <td>왼쪽 피연산자가 오른쪽 피연산자와 같거나 작으면 <code>true</code>를 반환합니다.</td>
+      <td>
+        <p>
+          <code>var1 &lt;= var2<br />var2 &lt;= 5</code>
+        </p>
+      </td>
+    </tr>
+  </tbody>
 </table>
 
 <div class="note">
-<p><strong>참고: </strong>(<code>=&gt;</code>) 은 연산자가 아니고, 화살표 함수(<a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions">Arrow functions</a>)를 나타내는 기호입니다.</p>
+  <p>
+    <strong>참고:</strong> <code>=&gt;</code>는 연산자가 아니라
+    <a href="/ko/docs/Web/JavaScript/Reference/Functions/Arrow_functions">화살표 함수</a>의 표기법입니다.
+  </p>
 </div>
 
-<h3 id="산술_연산자">산술 연산자</h3>
+<h3 id="arithmetic_operators">산술 연산자</h3>
 
-<p><a href="/ko/docs/Web/JavaScript/Reference/Operators/Arithmetic_Operators">산술 연산자</a>는 숫자값(리터럴 또는 변수)을 피연산자로 갖고, 하나의 숫자 값을 반환합니다. 기본적인 산술 연산자는 덧셈(<code>+</code>), 뺄셈 (<code>-</code>), 곱셈 (<code>*</code>), 나눗셈 (<code>/</code>)입니다. 이 연산자들은 대부분의 다른 프로그래밍 언어들이 부동소수점 값을 연산하는것처럼 동작합니다. (0 으로 나누는 것은 {{jsxref("Infinity")}} 을 발생시키는 것을 기억하세요) 예를 들면:</p>
+<p>
+  산술 연산자는 두 개의 숫자 값(리터럴 또는 변수)을 피연산자로 받아서 하나의 숫자 값을 반환합니다. 표준 산술 연산자는
+  더하기(<code>+</code>), 빼기(<code>-</code>), 곱하기(<code>*</code>), 나누기(<code>/</code>)입니다. 이 연산자들은
+  대부분의 다른 프로그래밍 언어에서 부동소수점 값을 연산할 때와 동일하게 동작합니다. (0 으로 나눌 경우
+  {{jsxref("Infinity")}}를 반환하는 것에 주의하세요) 예를 들어,
+</p>
 
-<pre class="brush: js">1 / 2; // 0.5
-1 / 2 == 1.0 / 2.0; // this is true
+<pre class="brush: js">
+1 / 2              // 0.5
+1 / 2 == 1.0 / 2.0 // 참
```

Comment:
```suggestion
1 / 2;              // 0.5
1 / 2 == 1.0 / 2.0; // 참
```
원문에 세미콜론이 있어서 넣어봤습니다